src/filter.ts

```diff
@@ -0,0 +1,46 @@
+class AttributesFilter {
```

Comment:
```suggestion
export class AttributesFilter {
```

By doing this, you can avoid that last line, `export { AttributesFilter }`